bounties_api/static/apischema.yaml

```diff
@@ -719,31 +1467,19 @@ definitions:
       paysTokens:
         title: PaysTokens
         type: boolean
-      old_balance:
-        title: Old balance
-        type: string
-        format: decimal
-      balance:
-        title: Balance
-        type: string
-        format: decimal
-      calculated_balance:
-        description: Does the token decimal calculation to give the base denomination
-        title: Calculated Balance
-        type: string
-        format: decimal
+      experienceLevel:
+        description: 0 -> Beginner, 1 -> Intermediate, 2 -> Advanced
+        type: number
+        enum: [0, 1, 2, 3]
```

Comment:
Hmm - what's the 3?

Reply:
this is a good question... It should not be there.

Reply:
fixed it here #58b6124c